src/gleam/erlang/file.gleam

```diff
@@ -109,8 +109,97 @@ pub type Reason {
   Exdev
   /// File was requested to be read as UTF-8, but is not UTF-8 encoded.
   NotUTF8
+  /// Path is not a directory
+  Badarg
```

Comment:
Let's use `Enotdir` rather than add this catch-all error variant.

Reply:
Turns out after diving into the Erlang source, I don't need the error at all. It's impossible to pass an invalid arg to the calls